src/CSI.IBTA.Administrator/Views/Home/Index.cshtml

```diff
@@ -35,18 +38,23 @@
         </div>
     </div>
     <script>
+        document.getElementById("employerBtn").setAttribute("class", "left view-btn-active");
```

Comment:
the recommendation is to extract js code to a separate file and execute it as a handler of onload https://www.w3schools.com/jsref/event_onload.asp